### ISSUE TITLE
Guide book

### ISSUE THREAD

**Issue by majinboyo** [WayofTime/BloodMagic#1474]
#### Issue Description:
Is the guide book not complete yet? I don't see an entry for the ritual of the satiated stomach or the array of attraction. Also, some of the book pages say format error. Have these things not been added to the book yet, or is my book not displaying the entries for some reason?
#### What happens:
I don't see entries for some items and rituals.

#### What you expected to happen:
to see more recipes.

#### Steps to reproduce:

1. 
2. 
3. 
...

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: 1.12.2-2.3.3-101
- Minecraft: 1.12.2
- Forge: 1.12.2-14.23.5.2768

**Comment by mywither** [comment]
The guide book, same as the mod is indeed incomplete. Way wants the entries to be interwoven with the lore, you can have a story and get your information at the same time. However, until all the features are implemented and their specific way of functioning is solidified, the guide book may lag behind a bit. It would be a pain to interweave the lore and story of the mod into a feature that was removed or changed in some way for balance that makes the entry and lore outdated. Writing out the story takes a while, especially when bug fixes and testing are going on at the same time. I do hope that it gets some more content though, so people know what's available in the mod: a good few things have yet to be added, like the specific effects for the Penance of the Leadened Soul downgrades and Living Armour upgrades, alongside the aforementioned Attraction Array, and some rituals like Le Volcanus Frigius.

TLDR: The Sanguine Scientiem is in an incomplete state, and needs updating, but that may not happen soon until features and specifics are implemented and fully worked through so as there is no need to correct misinformation in the book's interwoven lore.

I noticed the formatting error thing myself as well, but figured someone had already reported it. I think I noticed it occurring on a specific page in each entry, is that happening for you?

**Comment by majinboyo** [comment]
yes, it happens in specific pages. Leaving this aside, is there a way to learn about all the items other than the book?

**Comment by mywither** [comment]
Well, JEI is supposed to be used alongside the book, and it provides helpful information for not only recipes, but also has integration for the Penance of the Leadened Soul on what each downgrade costs and for what level of that downgrade. The Living Armour should for the most part be self explanatory with the upgrades, if you have questions, https://ftb.gamepedia.com/Living_Chestplate is decently comprehensive with a few minor errors. I find that the Developer Commentaries have been really useful for figuring out what some of the less explained things do, even if some of the information has changed, like the Demon Will Crystal Cluster growth values and how one gets the different Will variations. For Ritual Augmentation, you can make a ritual use a type of Will, and then it will show the Will augmentation information in the Information mode (Or at least you were able to in 1.10.2, not sure about that in the current iteration of Blood Magic 2). Other than that, just play around in creative and you should get a pretty good idea of how everything works.

**Comment by majinboyo** [comment]
thanks

**Comment by MentalMouse** [comment]
Also, the Alchemy Table can be explored (only) through JEI.

**Comment by keraldi** [comment]
That's exactly what JEI is for.

**Comment by mywither** [comment]
lorce, how to craft something and how something works are entirely two different things, so the guide book is still very helpful. Getting recipes, like for the Alchemy Table mentioned just above, is the purpose of JEI. However, you can't craft a ritual, you build it, augment its effects, change its area, you craft the parts to make it, but you don't craft it in itself. You don't craft an Attraction Array or a Requiem of the Satiated Stomach. Nor does JEI explain their purposes.

**Comment by keraldi** [comment]
The purpose of ritual building is fulfilled by the Ritual Diviner.
Everything else (if it is not covered by the guide book already) is work in progress.

If you feel the urgent need of having that (in your opinion) absolutely necessary information provided, feel free to contribute to the project or update corresponding third party wiki articles.

Open source lives from its community. Sooner or later we all die. That includes the owner of the project (WayofTime), you and me. So if you want Blood Magic to survive (and hopefully thrive), people will have to pick up some sort of work.

Feel free to ask questions about how stuff works in issues or on the Discord server (https://discordapp.com/invite/VtNrGrs), I'll gladly explain everything to my own ability for you to explore the mod or expand the documentation.

**Comment by mywither** [comment]
I'm saying that, I like when the stuff gets updated and in my last post here that JEI is not a fix-all explanation mod (the Ritual Diviner, as you said, fulfills the purpose of Ritual Building, not JEI), not that it is necessary for information updates right now. I understand that things are WIP, and said so to answer the original question.
As far as putting the information I know out there, it poses the same issue of becoming outdated as the mod updates that the Sanguine Scientiem has with its lore interwoven info. Just the same, I may have things I misunderstood and I don't wish to mislead people. (i.e. I for a while believed that the 1.10 BM2 had Steadfast Will drain at any amount in the aura to grow its Crystal Clusters, when I instead forgot I had a ritual consuming Steadfast Will in the chunk and it was just growing from the Raw Demon Will.)
I guess one could call me impatient to want the features to be finished up, solidified, and be able to continue reading through the story and lore of the mod now, but it's not quite accurate as I can indeed wait for it, just simply want to have it been finished already. The authors and contributors are only human, and I joined in loving the mod while it was in development.  I will simply have to wait to satisfy my want to see and play the finished result.
If I misunderstood anything, feel free to explain further in the Discord, I'm sorry if I didn't properly address anything.
